packages/core/src/browser/view-container.ts

```diff
@@ -837,6 +857,20 @@ export class ViewContainer extends BaseWidget implements StatefulWidget, Applica
         return false;
     }
 
+    protected getSashHoverDelay(): void {
+        const sashHoverDelay = this.corePreferences.get(SASH_DELAY_PREFERENCE);
+        document.querySelectorAll<HTMLElement>('.p-SplitPanel-handle').forEach(handle => {
+            handle.style.setProperty('--theia-sash-hover-delay', `${sashHoverDelay}ms`);
+        });
+    }
+
+    protected getSashSize(): void {
+        const sashSize = this.corePreferences.get(SASH_SIZE_PREFERENCE);
+        document.querySelectorAll<HTMLElement>('.p-SplitPanel-handle').forEach(handle => {
+            handle.style.setProperty('--theia-sash-width', `${sashSize}px`);
+        });
+    }
```

Comment:
It should be possible to find where this value is defined in the style in the header and set it once there, rather than setting it separately for all the handles. Alternatively, the rules that set this width could be managed in a dynamically created `style` element, and not written into a CSS file at all.

Reply:
@colin-grant-work Thanks for the feedback! Since the `sash`/ handles are defined as a pseudo element (**::after**) in CSS, I was only able to change their UI through TypeScript using CSS Custom Properties. I was wondering if there is a better approach for making UI changes to the pseudo elements.